src/proof_of_possession.rs

```diff
@@ -11,6 +11,15 @@ use url::Url;
 
 const JWS_TYPE: &str = "openid4vci-proof+jwt";
 
+pub type ProofSigningAlgValuesSupported = Vec<ssi_jwk::Algorithm>;
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+pub struct KeyProofTypeWrapper {
```

Comment:
I would rename this to `KeyProofTypesSupported`. Maybe the `Key` prefix can go as well, I'm not sure where it was coming from in `KeyProofType`